<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manualclassifier</class>
 <widget class="QDialog" name="manualclassifier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manual classifier</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>mitoticindex.png</normaloff>mitoticindex.png</iconset>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>478</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="interfasebutton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>interfase</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mitosebutton">
      <property name="text">
       <string>mitose </string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="notcell">
      <property name="text">
       <string>not a cell</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="unknownButton">
      <property name="text">
       <string>unknown</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="unclassifiableButton">
      <property name="text">
       <string>unclassifiable</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveclose">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>save and close</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cellimg">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="numcell">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>cell number</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
